Source/Synthesis.Tests/Fixtures/ContentSearch/FieldTypeTests.cs

```diff
@@ -148,7 +150,10 @@ private void ExecTest(bool useStandardFilters, Action<IQueryable<ISearchTemplate
 			{
 				using (new InitializerForcer(new SearchTemplateItemInitializer()))
 				{
-					testBody(context.GetSynthesisQueryable<ISearchTemplateItem>(useStandardFilters));
```

Comment:
This should be unnecessary once we add a DI model to GetSynthesisQueryable (see https://github.com/kamsar/Synthesis/pull/21/files#r32533998)